starter-code/iteration-1/main.ts

```diff
@@ -1,11 +1,28 @@
 // Iteration 1
 // 1. Create an array of strings
+let arrayOfTasks: Array<string> = [];
 
 // 2. Create a function to add a task in to the array, call it `addTask`, the function has to accept a string and don't have to return anything.
-
+function addTask (task: string): number {
+    arrayOfTasks.push(task)
+    
+    return arrayOfTasks.length
+}
 // 3. Create a function to list all tasks, it must show in the console de task.
+function listAllTasks (): string {
```

Comment:
You don't want to return the array, only showing the elements in the console. Then, the function will return 'void'.